src/CommandingOfficers/AW3/AW3ValueFinders.java

```diff
@@ -0,0 +1,73 @@
+package CommandingOfficers.AW3;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.IValueFinder;
+import Engine.UnitActionFactory;
+import Units.Unit;
+import Units.UnitModel;
+
+public abstract class AW3ValueFinders
+{
+  public static class CostValueFinder implements IValueFinder
+  {
+    int maxDamage = 3;
+    @Override
+    public int getValue(Commander attacker, Unit unit)
+    {
+      // Ignore Oozium, if we ever add those
+      int health = unit.getHealth();
+      if( health < 10 )
+        return 2;
+      health = Math.min(health, maxDamage * 10);
+
+      int value = unit.getCost() * health / UnitModel.MAXIMUM_HEALTH;
+
+      if( !unit.CO.isEnemy(attacker) )
+        return -value;
+      return value;
+    }
+  }
+  public static class HealthValueFinder implements IValueFinder
+  {
+    int maxDamage = 3;
+    public int getValue(Commander attacker, Unit unit)
+    {
+      // Ignore Oozium, if we ever add those
+      int health = unit.getHealth();
+      if( health < 10 )
+        return 1;
+      health = Math.min(health, maxDamage * 10);
+
+      if( !unit.CO.isEnemy(attacker) )
+        return -health;
+      return health;
+    }
+  }
+  public static class CaptureValueFinder implements IValueFinder
+  {
+    int maxDamage = 3;
+    public int getValue(Commander attacker, Unit unit)
+    {
+      // Ignore Oozium, if we ever add those
+      int health = unit.getHealth();
+      if( health < 10 )
+        return 1;
+      health = Math.min(health, maxDamage * 10);
+
+      for( UnitActionFactory action : unit.model.baseActions ) // Does it make sense for this to be dynamic?
```

Comment:
If we have or ever intend to have units that can capture but only sometimes, then yes.
Also, do we care about capture power? Might want to feel more threatened by Sami's infs than others. Though you can't really factor in her SCOP capture power, since if you can see that it's active, it's too late to stop it. I guess you could also poll the other CO's star meter...

Reply:
We don't currently, and it's a very jank design space that I dunno if I like.
No. That has no basis in the cart logic.
That... also has no basis in cart logic, but I have very vague plans to use the mod enqueue hook at some point to let AIs plan around powers and/or implement a fancy damage calc screen.